modules/openapi-generator-gradle-plugin/src/main/kotlin/org/openapitools/generator/gradle/plugin/tasks/GenerateTask.kt

```diff
@@ -616,9 +623,16 @@ open class GenerateTask @Inject constructor(private val objectFactory: ObjectFac
         }
 
         inputSpecRootDirectory.ifNotEmpty { inputSpecRootDirectoryValue ->
-            run {
-                resolvedInputSpec = MergedSpecBuilder(inputSpecRootDirectoryValue, mergedFileName.getOrElse("merged")).buildMergedSpec()
-                logger.info("Merge input spec would be used - {}", resolvedInputSpec)
+            var runMergeSpec = true
```

Comment:
suggestion: can you please add comment explaining what runMergeSpec does?

you can copy from the description of this PR so that other contributors later reading code understand what problem/issue it's trying to solve.